server/controllers/UsersController.js

```diff
@@ -84,6 +84,83 @@ class UsersController {
       }).catch();
   }
 
+  /**
+   *  Get all users
+   *  @param {Object} request
+   *  @param {Object} response
+   *  @return {Object} json
+   */
+  static getUsers(request, response) {
+    const userId = request.token.user.id;
+    const query = `SELECT * from users where id != ${userId} AND status = 1`;
+
+    client.query(query)
+      .then((dbResult) => {
+        if (!dbResult.rows[0]) {
+          return response.status(404).json({
+            status: 404,
+            success: false,
+            error: validationErrors.noUsers,
+          });
+        }
+        return UsersController.usersSuccess(response, dbResult);
+      })
+      .catch();
```

Comment:
#### Critical
No `catch` implementation. Why?

Reply:
I have implemented the feedback